ROLES.md

```diff
@@ -272,3 +285,13 @@ Vendors contribute extensions to the Istio project in the form of new adapters,
 
 * Each vendor receives access to a subdirectory in the contrib repo, and each directory has got a distinct OWNERS file granting
 approver permissions to the vendor.
+
+## Sponsor
+
+A sponsor is a member that wishes to vouch for another user to become a member.
+
+### Requirements
+
+* Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating on issues, etc.
+
+* Sponsors must be reviewers or approvers in at least 1 OWNERS file (in any repo in the Istio GitHub organization).
```

Comment:
Do we want to say something about sponsors being responsible for the conduct etc. of the prospective members? Basically make sponsors understand the responsibility in sponsoring someone...

Reply:
Added a bullet for that.